src/main/java/com/bloomtech/asylumrgbea/entities/AsylumCase.java

```diff
@@ -37,4 +37,16 @@ public class AsylumCase {
 
     @DynamoDBAttribute
     private String asylumOffice;
+
+    @Override
+    public String toString() {
+        return "AsylumCase{" +
+                "id='" + id + '\'' +
+                ", citizenship='" + citizenship + '\'' +
+                ", caseOutcome='" + caseOutcome + '\'' +
+                ", completion='" + completion + '\'' +
+                ", currentDate='" + currentDate + '\'' +
+                ", asylumOffice='" + asylumOffice + '\'' +
+                '}';
+    }
```

Comment:
Double check the variable names, I thought some of them were updated, but maybe that will be done in another ticket.